exportReport/Utility/SpringUtilities.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 1995, 2008, Oracle and/or its affiliates. All rights reserved.
```

Comment:
In this case I think we just need to bundle this license notice in the add-on.
There are already other add-ons doing that, for example, in [pscanrulesAlpha](https://github.com/zaproxy/zap-extensions/blob/alpha/src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml#L53-L54).